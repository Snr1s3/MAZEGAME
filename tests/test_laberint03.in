gAAAAABmFvgEt70HIIIXmayq3-guXsyC0vI3mqqkk8clV5PrixOzaNENBP2oRKZ8xGPEuV_ITupr60UutX3p8sOctp9yV2C-cQrzX_2T5uge0bYz0ik7wTTrIv8y4mBWgO_3ds37G9nr