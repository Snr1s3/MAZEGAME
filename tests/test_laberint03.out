gAAAAABmFvgE03D2_CV5uMXrfhoAwiEZGSPP8EFRFt_icwdh0rEaQO-Qrzph_VXNbwV55b3PKD8XTIaoROwwaohMx38MryZZFQWwD_Xj9yfyo7eJ9L1yuXVZfY6o0TMyheu4E7Zqitb5oFOsSYcEVha0sifBCDgq0oKLV6dJkohTM4oK8-txo2Ue38LgAwg-eMbCQdfbluFJzQdHUWmic73Zfb1yemyhQomsAbHFE2Y_N0LdSnXObRdTQ5_6dQasfb_qI4kYMBcOEiPZocObjpEmrmX7AcF4Qdv--CxBCU1TntKZn6tcigMY-qA0aQebg16kMs8N53oUJgfB46g52gsyMibE7BzYixqtY5RX_zLa_Cxx_L_RbTlwcTW2HQOJmg2_dNGk7eTNqEujdmZWtkmt_zDSHj6lPlJwYNAB8mltHpHEvzp6W7I3ApZoXGevJaq6YChmXo3PetFp6L_U7wTBi1ZoAmbom_InAVorjZCSWuug_aW5jKaVyNR1oG9bnS7NOhSsRLkbjmjh8O83GdS622VEP6xr9fNtj2Dcojhb8MGRtGZ-8txk_2aN4oOPZu49YimaYnjXFS8GNVj0cSrQOoh9q5SJu5CRT1mToMamASfgSZlE7X0JppQ_ZI-gUt-M3eQZwoYJs99sl-6heiqqMuzdyzKVeKrvqXBJUcQ4Bq2bbVJJHNx8VftlA_5RJn5QrZ6nQKETmwNrqfvT-7lXkQAJ_TxGwe3W8k8lfd8yeF9I30Fw_RiGAE_dErY_TqGpp_7kMeaS0bT8y4sN7Bno5g50oCqxNbIhT257J0PChHCRVaK1_JtNaASuSTGOXyfAGumqJzkvvQFOSQWNuccNm5N2o4BEQWSDEsjcbNUx89XvlBorVWTVI5VUyuMw1hGGX48DaJImZeCwxHaOrTh4cRLBaf7BAk3A2ZOMP7vyKjz3uYSsW2_m-LbzCVHAshzoMQfuZIjaJLRfKjoKamZEijodNh80ujdOU1j3JMI_F0EGmDI_ZE7qF6nNc0RBqnwXi1LXpZWR-cj-b_YidGDyFBeUS0X6CicrUkUBhyd02sx6V737JyaMPldBC-Ur1BRL_beinUT7jeiq6OZ1EB_eRDrwTticHgvSiecfizIqKOkgnfzLC4Q22wg9gBWeqJmF6frvOBhyuDnoe_EO1Aol1-vYIk3It5qStET18oO5BaNKVu07FRHRYKcx-_kUaeQZvTlcg_SgN0Jx_kgqGat7ON9ZPQ58YZKpvqkwUffGpyFp_chJQNyuvu90DQjEwvLfdpgylnbofIEoWwIWDQ8r90--d87AFGTaFTgU4B1Z__A8bQpOAiIc-kQM2TY_T-5DZner8zuwGJ7d7-m2CaNqRumCJWvFYufGp36ZbY55qjTN5k16dFtwU5dXR9vhR2YAyMMoeltvdms9kStDENHoWBT_80GieqEZYF1z7OzML--Y4yL_IZFaVK6_M2i6BjTj51S23GsrMjR1z0sYxCQjBcR40i22nl1vLg0GfTH-sjWuxgCwPWku6XvgulZAJqXRCJyIIoJE3p6Ov07TMJPuRHP8Iv5840uZ8-3Pyf1EHvsZ3acqi9fmGXue2obC8iPDOGnLwxhfi_FE64zfv23cJKPFpxhKECgIspz5MA63QDgbVVxuDkeJhpjEWAAm4YdYmz6Q2qbqcU15uWReigwqf-LS5fuxWeIPtrzR7NgiBERVaUJJ7Sj2uB-jzdNww585o94LVOq6W93yE0eO1OlJOLdLSp8HPEIm9Mp7r17nLMLrEIqEktHpk_q8IDDi2MElgi4eDP3kv4pI80tjVsfjd8u-R8JEAI2gz1xPrfZmreIfdyeC3xq5sMshrD75wiwgj-4xB4EM2xUXhcae8oZO7UmAhdhBOzKCqmcUhXpQMdHj0U7HbzjqgWFKBps9ZnGUSLjXzRTDlfLSU1ykmJU3K7nXfU6D6b0eJpSHYjQyPnmJlmbaRCjlRpwFIgOijNCQoXjFy9V4-MnhGHAZxRoP1mLXlVXB8pITfVePO05bZ2QDn5YEQTrVEP9v1YHGp463K-YR8HSmLLHgQXh_xTw-0jOdG9DlcXJPQAEwDGuUoseOzOwwZJxN5Gmhu16EuxLqp2eQbYOWaOryGHM3a9UEaLYTyCwQ80th6gdbiPTlG5DlIXhO5XdKxB1_8I1ooAQ7sQOa2YRisWWEnBsO4U36pQE1wfLB2vqhTtoAuRh0IuDrHgEL3ZJDoQOjhtw3RYRs2LrMHYXWGtB7kJ-hXa8hC6oKlxUS5r4p5ERoDI34DSMDO5zbHUoLeXRKZBHcCwXjKfI-vy_lIH5GBIZnc1ty1UiAmDDtnasPyAyWysF6Z8kkrmUZt0b2kM1qptsG5UnIxLgfANlka1tAqsPMWi26vfvHzekQ_70qR-vw1VTokuTHexg_ZTIavrCrZcHgiVBpZJh9UW07pVemyx5cSV0_PIqCDyfzX25myFghiwyHQRXF6WiXRp3_HRTVJWX9dr7XJ4VVf0igSgePvz7EZIifBsl0r7h-TM1j75Kf1hSzz-anRO_BzhMiZTnugurRcNxTG_Ecq0R0aDJMOYnwoKxl5k_QD_6gFGCZsON8RJUukX9a_PtJw57fjcv9V52_Y6t7ojZ0cqpFZyoPUpxrZnb8GB67pmYvVIcNknBy7nl62tpR59uym5wEtGr5NBo4E_F0ICOS1kh6B_hXfVDaQRlZNWYNAg-0UE-S5C_dJtkOg7tx59d10-aeBZLmkOZW4o-B6DWRY33l68s0mMehvfTnoJe31lSmXJrDuZ2aDDmvo2beSqTYSel_HMEI4BwrTP17Zo2X3B7R9SzeWfsB6Q2dnW3b8JcpRlKEyChr2adiQ4uLM5MebaA-Z6mNKEEJBQPF_wVCJ3grB_Rfeb3iXkJb16XsPVELg5Iaj-zpWXTEGDWTlAYxzcOUdpwLKoID5arUSWX0Bvd8Ukxo_GI1aAye9h7Tw4sKIjftw8kvtKYNE9TAnpqgLPCIkknjfKP-ke_eLRQyJvUsfT1Ac7X34vqHyyaYXaPxGAkL-LCL-Y6jyeOupm7cdN5OAW0AZdk7UzZGItTvn4TQnHeMRCzJccPx0rkN4qNzGX9d8CVtA1qsVR3cXXmucdtOcuAVZQlj3T2TOKC28VoPJ-y86EikdGbuYT3pcnWZ7kqvNGZVanyY1g5LVnlyJmFKovQ_ZgYuRpG4BdNYu4Jtbt5n-pI_aDGsCj2BnglP4uNM5LRIElMjwVI6zD5DR_RTRbBGJkSdQgcjO4dfEgGV4HxKh6Kg9ggnn04OtgAdyYVUt_blM6ZgkZq0JUP0JwF0kkWEcDJuTuUXW7g6wYM-H-4YA0JPLPBzss62fP3l_luKeh8d8QKIyXHCckdFUlTdl-Jk6PDo7j8TrJkTVKHOI0ikUKJezPLmmU4tUUMJrSw-cBAvmrmU-qdWmSIXTGRMD6RiS92EsN7PDq-1-oA1eaxW8FUp4jvsm34pZofWdA0Bg4tUnjboAwVVrga4BCBEsnJnTz3xu-jK431VYC69htdK34QwJi1MNPLfIIQI68a5AUXgKyNVyNVwGZWv5J5WRUMlDPm9OsFpvcX_LHsUVSj2pYE3QG29TZlicrab0f_gsWtJoOX5Yak23k7vF-wepDU6o9-KXDKDTnr1kButmSIpE33Zc5Gcub-LTyRv11J7KEOlSkC7Jt8kppjZFfmq068iJ4g74o0nLIg9l2d9UtilBf7Q-WdEyfARYUMW18YIWOzyyIoaz81rJDaUpAvPbRD-UuR8R1Iu0dgnaWtNfhpJkA63DNZKRhvQ1m0d_J6zqMxPLXq83C6BF19heM5-owVe099iMTgymWPKcRkwHMc5__oNQ0GlBHR7Uj-tQgYa-a3-Okcb94LciZ2vQZD3NKV4Yt_bpXtJkpY7_h3zfEmcndiHxVMR8BeR-ESZaBA5qCV90k_fv0ay_2eS1UwQHkeiX_HrOUvngNNEeGqcL7tqcQGBEoaoJFYy5J-NObQvA-shhb4rnTPj-JEHnffShOph38GMZ4Ow9Tef9ZaTYftLS91Yfv2hwzFcNZ3HX3TUQAU1cU8S2N809_HFsctJcunVoysh_U33_iRVxUjnBCvOC9BOK_8GfL20KRLY6i-zHjr82NgVtLRDwLyDUzu3m3DmOjTbasxmVgLzPzbhboUG8ITtVTQWQjV4YL2w42j1am4Cug86HVVS9F1Vj8O5dih-orKW9-Y43bWdi2DoeA-fEv97zO1VMpZLz5pd21CZb1QD3NcP8OcsqRKpu6jI3gx7zgEjZmwusCVITz1En_gwL_nBHIseMLdaVm-HqX1M3GlAHI4MZIKu928npeTb3BU3zfPm9UOL07s1Dep4XPWuVQ3S2qMZNN2F9Qie2LtGxOLmdJq4hgmvYGRo_TNYd6hPvIAKHkPM0rWCZEJFnCt9uQoveNpceuMzat0oIMBLKgeFGYsbdeiQM9MMUZq9Mp-jB5a-sOAAPzY3p3t0427LFHZa0PWmiZ7v-Am-HC5fSc6Jh1HjCf5OmRIllMYB0B1oD2-jU80JGiUu5S-gaXMciimuhbeLEd-ZtkpXgHM2BWjFNjNWLburO3YOmuQ_YFd52bkz1RNQ4YjxmKnHjErzkDpx0pxC3rAZoaVM20SWe0JDEKbcVSyur8weTgbnn8Zm0K-42UTv7hiJz8M921b-7RlFl__oPbXWYKc_zyXdQeFjjWzqohw7oRhQoJq1dmX1r0UC-KxwPbOC-7uVCqxj_9NTUCbFuviZM0AdMXjWATXqsCPczs-dGM5gmbg_pIivSxfwXVkb4hQjfmnMQuVqv9Ang8ZsMGbSfeSs-ZI6aQgQNplDnQ4SP-hxyI9au7ChKkP3vT5_g2J-5RDdAYARwJKciAMh_SsMVRVyWNAPedzsEEbwvy0PlGgVB0nqBhh5UGTXzmazy4CkVIgH7QUSYr1VQ_2uKvE4dXE8s4ItW9M2k6V0j_rQUypwikjfA7aoae4nJdFJpKCeEhpswIPnTuhPSnGnPiaOxF8A-_HhrC5qYS30Sr5y8c6pBIrdDZbCApeJxSncngj_5-XRasK95c3bq-u9Y_CIF4ndD_YKTWQ0cGn-n01Uz6XdLexw7DwieRbX7fc5FMUs7L1yUrLHgrs9K6B6E3y3GShUC-vSbo2jyVThPX48qr0vGcfcAQIWS4JmSsJ-xbhn9K-Fl7PkZ-WnfeQ9crgllQXRew_Ee-Va6ELALYqVrLOS5DxrBLOVPcNq-pvQCiVgoAaW9X73qWpRbG7QlTXNzwHfcVhJHNJhxPtGN8oaZLUeKA4upiorRbQEhUIrK0KuCgqwYM9xvnVUXIQ9hIQTeeOWOD1wuXoI5Kzzsd0BnyHYIlOcsKzUvlnTcqjmjGF73iMr4ufgo53mHYHXWNeYzLy3pQ6f6ttsVRXOY0Il1uy33eblSohbo4Qg5o1a_zZ7EpEIdfIIBMikbO8NBvD3SJSnfv64sWlVp2NKl5Rht6XcIHvLk4ubrdp_YCCqqAlO6W0DITwtdtf9zofPi9EsCtGTjNt6et8m6LWmnb669NNm_PDjn7jH-wb5EPy651UDPSnp84NbHdYwxq4hp11K9KMiGVx4DiMaaI-ddW_XUIQ3eeUOEesKpz8einoHR42K73t4UbFKM6dWSwmcHBmLPHwt5v0ctNbOvM1vJZGoeJDg-1bzgOu_VaDkfbZ0daNtB_We1qurFb-Uiwq04nQtC5DvbGf9qhOInQ-HqzUkKTKVHJ2tNPjpELzMq2hxhk7zfjNxXEEoDCzSMtTgD5dJ_pZON387tyre2IVXCrl6SzkVKp5Flc3iMQKmF7LetYNLF4Iu